api/server.js

```diff
@@ -20,14 +16,88 @@ mongoose.connection.once("open", () => {
   console.log("MongoDB connected");
 });
 
-app.use(
-  "/graphql",
-  graphql({
-    schema,
-    graphiql: true
-  })
-);
+const Location = require("./models/location");
+const Venue = require("./models/venue");
 
-app.listen(5000, () => {
-  console.log("api running at http://localhost:5000/graphql");
-});
+// GraphQl type definitions(similair to models for db)
+const typeDefs = gql`
+  type Query {
+    getLocation(id: ID): Location!
+    getLocations: [Location]
+    getVenue(id: ID): Venue
+    getVenues: [Venue]
+  }
+
+  type Location {
+    name: String!
+    state: String!
+    id: ID!
+    venues: [Venue]
+  }
+
+  type Venue {
+    id: ID!
+    businessName: String!
+    category: String!
+    website: String!
+    phone: String!
+    address: String!
+    locationId: ID!
+    location: Location
+    state: String!
+    zip: Int
+    hours: Hours
+    parking: String
+  }
+
+  type Hours {
+    monday: [Int]
```

Comment:
I don’t see any keys on here. How will we connect them to a location?

Reply:
This is part of graphql i'm still trying to figure out and may change depending on the data structure our form sends. But i was trying to create an object "type" to be a field of another type. In the end should all be saved as one document to db